terraform/modules/cloudfront/distribution/distribution.tf

```diff
@@ -0,0 +1,56 @@
+# Terraform module for a single CloudFront distribution with Shield Advanced enabled.
+
+locals {
+  originID = "${var.origin.lb_dns_name}-origin"
+}
+
+resource "aws_cloudfront_distribution" "distribution" {
+  origin {
+    origin_id   = local.originID
+    domain_name = var.origin.lb_dns_name
+    custom_origin_config {
+      http_port              = var.origin.lb_http_port
+      https_port             = var.origin.lb_https_port
+      origin_protocol_policy = "https-only"
+      origin_ssl_protocols   = ["TLSv1.2"]
+    }
+  }
+
+  enabled         = true
+  is_ipv6_enabled = true
+
+  aliases = var.aliases
+
+  restrictions {
+    geo_restriction {
+      restriction_type = "none"
+    }
+  }
+
+  default_cache_behavior {
+    allowed_methods = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
+    cached_methods  = ["GET", "HEAD"]
```

Comment:
Do we want to have any caching on this Cloudfront? This distribution will be protecting all apps on *.app.cloud.gov, which can vary widely. If we implement caching, I fear that we might have difficulty with needing to clear the cache for some sites but not others. 

Would it be better to disable caching and just do a pass through to the load balancer?

Reply:
Per this & other discussions, I'm going to remove caching from the distribution 👍